pkg/registry/service/rest.go

```diff
@@ -276,3 +300,21 @@ func (rs *REST) deleteExternalLoadBalancer(service *api.Service) error {
 	}
 	return nil
 }
+
+func externalLoadBalancerNeedsUpdate(old, new *api.Service) bool {
+	if old.Spec.CreateExternalLoadBalancer != new.Spec.CreateExternalLoadBalancer ||
```

Comment:
Should this also check (old.Spec.CreateExternalLoadBalancer || new.Spec.CreateExternalLoadBalancer) ?  If neither spec has a load balancer, you can safely say "false" right?